src/cli.ts

```diff
@@ -14,6 +14,11 @@ const cli = meow(`
 			Options
 				-c --context  Your AsyncAPI specification
 				-w --watch Enable watchMode (not implemented yet)
+		context
+			current                show the current set context
+			list     			   list all the contexts
+			remove  			   remove the current set context 
+			use <fileName|URL>     set/update context 
```

Comment:
looking at our discussion in the issue, we still need `context set <context-name> <filePath | URL>` right? and the `use` should rather be `use <context-name>`?